src/main/scala/play/modules/reactivemongo/MongoController.scala

```diff
@@ -124,23 +124,29 @@ trait MongoController {
 
   /** Gets a body parser that will save a file sent with multipart/form-data into the given GridFS store. */
   def gridFSBodyParser(gfs: GridFS[JSONSerializationPack.type])(implicit readFileReader: Reads[ReadFile[JSONSerializationPack.type, JsValue]], ec: ExecutionContext): BodyParser[MultipartFormData[Future[ReadFile[JSONSerializationPack.type, JsValue]]]] = {
```

Comment:
With a default value, these functions can still be called without passing a custom `maxSize`, so it doesn't break the API compatibility.

For the implementation of `multipartFormData`, it can be questioned on the MailingList of Play itself. That's out of scope for a PR on this plugin.